homeassistant/components/media_player/apple_tv.py

```diff
@@ -44,17 +44,23 @@ def async_setup_platform(hass, config, async_add_entities,
     """Setup the Apple TV platform."""
     import pyatv
 
+    if config.get(CONF_HOST) is not None:
```

Comment:
Change this check the other way around.

```python
if discovery_info is not None:
    # …
else:
    # …
```

No need to return `False` when reverting changes to `PLATFORM_SCHEMA`